bin/build-hdfs

```diff
@@ -8,7 +8,7 @@ IMPALA_VER=cdh5.3.1-release
 IMPALA_ZIP=$IMPALA_VER.zip
 IMPALA_URL=https://github.com/cloudera/Impala/archive/$IMPALA_ZIP
 NATIVE=native
-DIST=hdfs-mesos-0.0.2
+DIST=hdfs-mesos-1.0.0
```

Comment:
Let's just jump up to 0.1.0, not quite a 1.0 until we've added a few more features and gotten some in-production testing.